ctapipe/containers.py

```diff
@@ -221,6 +228,7 @@ class ImageParametersContainer(Container):
     peak_time_statistics = Field(
         PeakTimeStatisticsContainer(), "Peak time image statistics"
     )
+    core = Field(CoreParametersContainer(), "Images direction in Titled/Ground Frame")
```

Comment:
Same as above

Reply:
I missed it, thanks

Reply:
Ah also it says "Titled" instead of "Tilted"